diff --git a/node_modules/@getlarge/kratos-cli/src/app/login.command.js b/node_modules/@getlarge/kratos-cli/src/app/login.command.js
index b1c9284..e2abc4f 100644
--- a/node_modules/@getlarge/kratos-cli/src/app/login.command.js
+++ b/node_modules/@getlarge/kratos-cli/src/app/login.command.js
@@ -60,6 +60,9 @@ let LoginCommand = LoginCommand_1 = class LoginCommand extends nest_commander_1.
                 identifier: email,
                 password,
             },
+        }).catch((err) => {
+            console.error(err?.error?.response?.data ?? err);
+            process.exit(1);
         });
         this.logger.debug('checking session token:', sessionToken);
         const { data } = await this.oryFrontendService.toSession({
diff --git a/node_modules/@getlarge/kratos-cli/src/app/registration.command.js b/node_modules/@getlarge/kratos-cli/src/app/registration.command.js
index 385648f..235ccc2 100644
--- a/node_modules/@getlarge/kratos-cli/src/app/registration.command.js
+++ b/node_modules/@getlarge/kratos-cli/src/app/registration.command.js
@@ -60,6 +60,9 @@ let RegistrationCommand = RegistrationCommand_1 = class RegistrationCommand exte
                 password,
                 method: 'password',
             },
+        }).catch((err) => {
+            console.error(err?.error?.response?.data ?? err);
+            process.exit(1);
         });
         this.logger.debug(`Registered with email: ${email} and identity.id: ${data.identity.id}`);
         console.log(JSON.stringify({
